examples/literate/literate_gol_lazar_belta.md

```diff
@@ -0,0 +1,176 @@
+# Example: Gol, Lazar and Belta (2013)
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/egidioln/Dionysos.jl/master?filepath=examples%2Fliterate%2Fgenerated%2Fliterate_gol_lazar_belta.ipynb)
```

Comment:
Do not push this file, you can exclude `*.md` in the `.gitignore` to make it easier.